base/src/commonMain/kotlin/utils/OxygenationData.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+//Based on https://github.com/splendo/health-app/blob/monitor/SplendoMonitor/SplendoHealth/Domain/EventBus/Events/OutputData/Measurement/OxygenationDataSource.swift
+package com.splendo.kaluga.base.utils
+
+
+interface OxygenationData<T> : SciUnit<T> {
```

Comment:
This is Splendo Health data leaking into Kaluga. Sshould be removed

Reply:
Will be moved to a separate Kaluga module soon